README.md

```diff
@@ -1,2 +1,48 @@
-# cheesecake-api-tests
-API tests for Pantheon v2
+# cheesecake-api
+
+API tests for Pantheon v2.
+
+
+## Setup
+
+### Requirements:
+- Python3
+- virtualenv
+- virtualenvwrapper
+
+### Installation:
+- Install all the dependencies.
+
+``` pip3 install -r requirements.txt ```
+
+- Set the PYTHONPATH to your current working directory
+
+``` export PYTHOPATH=</path/to/your/tests directory> ```
```

Comment:
There is a typo in the variable name. It should be PYTHONPATH

Reply:
made the change.